tests/integration/session/test_timeout.py

```diff
@@ -29,3 +29,13 @@ def test_schema_defined_timeout_cant_be_higher_than_server(self):
         time.sleep(4)
         self.get(self.last_url)
         self.assertStatusUnauthorised()
+        self.assertInBody("To help protect your information we have signed you out")
+
+    def test_submission_complete_timeout(self):
+        self.launchSurvey("test_timeout")
+        self.post()
+        self.post()
+        time.sleep(4)
+        self.get(self.last_url)
+        self.assertStatusUnauthorised()
+        self.assertInBody("Your census has been submitted")
```

Comment:
As above

Reply:
Point taken, but in this case it shouldn't be needed as it's already checked it's a 401

Reply:
True, just the functional test then